spec/features/global_announcements_spec.rb

```diff
@@ -35,23 +34,21 @@
       let(:announcements) { build_list(:instance_announcement, 2, instance: instance) }
       before do
         instance.announcements.clear
+        SystemAnnouncement.destroy_all
         announcements.each(&:save!)
         visit root_path
       end
 
       it 'shows the latest announcement' do
-        announcement = announcements.last
```

Comment:
The <code>last</code> announcement cannot be found and I changed to <code>first </code>, something wrong with the implementation ?